src/data/general-cs-quiz.ts

```diff
@@ -1330,6 +1330,17 @@ const generalCSQuiz = [
       "Bubble sort has a worst case time complexity of O(n^2), which is slow compared to other sorting algorithms.",
     Link: "https://www.freecodecamp.org/news/bubble-sort/"
   },
+  {
+    Question:"Which Time Complexity is said to be good compare to other?",
```

Comment:
You should be carefully with the word "good" because that is often seen as subjective. Let's go with this phrasing 

```suggestion
    Question:"Which of the following time complexities is considered to be the fastest?",
```